docs/features/cli.md

```diff
@@ -2,18 +2,62 @@
 
 After installing DAG Factory, the CLI can be invoked using the `dagfactory` command.
 
-## Usage
+## Commands summary
+
+| Command | Args   | Flags       | Description                                          |
+| ------- | ------ | ----------- | ---------------------------------------------------- |
+| `lint`  | `path` | `--verbose` | Check if the given directory or file is a valid YAML |
```

Comment:
Not in this PR, but I feel it may be worth providing an option to ignore some files